src/jog/dto/create-jog.dto.ts

```diff
@@ -0,0 +1,35 @@
+import {IsDate, IsNotEmpty, IsNumber, IsObject} from "class-validator";
+import {Type} from "class-transformer";
+
+class Location{
+    @IsNotEmpty()
+    @IsNumber()
+    lat: number;
+
+    @IsNotEmpty()
+    @IsNumber()
+    lon: number;
+}
+
+export class CreateJogDto{
+    @IsNotEmpty()
+    @IsNumber()
+    readonly userId: number;

```

Comment:
`userId` should not be part of the request. This allow users to create jogs for other users.

Reply:
fixed in the next commit inshallah😊